### ISSUE TITLE
Can't connect to Unreal running on Windows from Linux machine

### ISSUE THREAD

**Issue by ivannson** [microsoft/AirSim#2422]
I am trying to connect `CarClient` running on Ubuntu to a simulation running on Windows (Windows firewall is disabled), using #1551. 

In the script that is running on Ubuntu, I am trying to connect using: 
```python
self.client = airsim.CarClient(ip="windows.pc.ip.address", port=41451)
self.client.confirmConnection()
```

However, it is not connecting and exits with `WARNING:tornado.general:Connect error on fd 9: ECONNREFUSED`.

Checking the network connections on Ubuntu while it is trying to connect using `netstat | grep 41451` I get:

`tcp        0      1 MyUbuntuServer-GPU:53992      windows.pc.ip.address:41451      SYN_SENT`

Am I doing this correctly?

Edit:
I am using ip address found in "View your network properties" -> IPv4 address as the ` windows.pc.ip.address`

**Comment by rajat2004** [comment]
Not sure about the correct way to check the IP address, maybe check with `ipconfig` also
Check the network adapter if the correct IP is being used, such as using Ethernet to connect to the Ubuntu machine while WiFi is also on
Just some suggestions from the top of my head

**Comment by ivannson** [comment]
@rajat2004 Thanks for the suggestions, I think the IP addresses are fine. 

Just to check, in theory, to connect between the two, I do the following:
- On Ubuntu, running Python API, set the IP address of the rendering machine (Windows)
- On Windows, running Unreal, specify the IP address of the rendering machine (so itself) in `settings.json` in the `LocalHostIP` field

Is there anything else that needs to be done?

**Comment by jonyMarino** [comment]
Closed as in #2386
